ada/ast.py

```diff
@@ -14800,6 +14870,24 @@ def expr_type():
 
     env_spec = EnvSpec(add_to_env_kv(Self.name_symbol, Self))
 
+    @langkit_property()
+    def is_constant_object():
+        loop_spec = Var(Entity.parent.cast(ForLoopSpec))
+        return If(
+            # IterType.alt_of loops are constant if the iterable object is
+            # constant. Two cases are supported so far:
+            # 1. The iterable object is defined as constant such like a
+            #    constant array.
+            # 2. The iterable object is not directly declared as constant but
```

Comment:
I think that now that you use `is_constant`, this comment is useless

Reply:
Also I think you're lacking the `Constant/Variable_Dereference` part of the RM, so that should be mentioned in a comment here

Reply:
I'm not sure such cases are missing actually since `is_constant` property is implemented for `ExplicitDeref` names. Are you thinking of something else?